### PR TITLE
README.md updates

### LINKED ISSUE #1108: yarn package fails with node 12.22.9
I have an Ubuntu system with node 12.22.9 installed, when I run yarn package I get this error:

> error @vscode/debugadapter@1.63.0: The engine "node" is incompatible with this module. Expected version ">=14". Got "12.22.9"

If run `npm install --global node@14`, then I get the error

> error @vscode/test-electron@2.3.8: The engine "node" is incompatible with this module. Expected version ">=16". Got "14.21.3"

Running `npm install --global node@16` then yarn package succeeds.

The README says Node 10 or higher should work, and CI looks like it only tests 16. What versions of node are supported?

Also, can the node version be defined in package.json or yarn.lock, or does it have to be installed outside of that?

### LINKED ISSUE #1136: README is missing from the published extension
When installing the extension from the VSCode Marketplace, the README is missing.

### PR DESCRIPTION
**Pull request by shanedell**
README.md updates:

- Add README.md to VSIX when built
- Update the minimum required node version for building.

Closes #1108
Closes #1136

### CODE REVIEW

File: webpack/ext-package.webpack.config.js [hunk 1]
```diff
@@ -101,7 +101,7 @@ module.exports = /** @type WebpackConfig */ {
   plugins: [
     new CopyPlugin({
       patterns: [
-        // { from: 'build/package/.vscodeignore', to: pkg_dir },
+        { from: 'README.md', to: pkg_dir },
```
Comment (stevedlawrence) [inline webpack/ext-package.webpack.config.js:104]: Should this be ```to: `${pkg_dir}` ``` like the others?

**Review by stevedlawrence — APPROVED** [review]
+1, maybe a typo?

File: webpack/ext-package.webpack.config.js (hunk 1 above)
Comment (shanedell) [reply webpack/ext-package.webpack.config.js]: Yup that was a typo, thank you!

**Review by mbeckerle — APPROVED** [review]
+1

**Pull request merged**